File: arm.64/langs/ocaml/src/trip.ml
let trip () = [
  { Type.address = "/version"; payload = "{}"; options = "{\"once\":true}" };
  { Type.address = "/storage"; payload = "{}"; options = "{\"once\":true}" };
  { Type.address = "sql.help"; payload = "{}"; options = "{\"once\":true}" }
]

File: arm.64/langs/rust/src/trip.rs
use crate::r#type::Triplet;

pub fn trip() -> Vec<Triplet> {
    vec![
        Triplet {
            address: "/version",
            payload: "{}",
            options: "{\"once\":true}",
        },
        Triplet {
            address: "/storage",
            payload: "{}",
            options: "{\"once\":true}",
        },
        Triplet {
            address: "sql.help",
            payload: "{}",
            options: "{\"once\":true}",
        },
    ]
}
